homeassistant/components/apple_tv/remote.py

```diff
@@ -53,7 +50,9 @@ async def async_send_command(self, command, **kwargs):
         delay = kwargs.get(ATTR_DELAY_SECS, DEFAULT_DELAY_SECS)
 
         if not self.is_on:
-            _LOGGER.error("Unable to send commands, not connected to %s", self._name)
+            _LOGGER.error(
+                "Unable to send commands, not connected to %s", self._attr_name
+            )
```

Comment:
```suggestion
            _LOGGER.error("Unable to send commands, not connected to %s", self.name)
```
It looks my cleaner here accessing the class property `name`